docs/static/releasenotes.asciidoc

```diff
@@ -76,12 +76,39 @@ This section summarizes the changes in the following releases:
 === Logstash 8.17.0 Release Notes
 
 [[known-issues-8-17-0]]
-==== Known issue
+==== Known issues
+
+[[known-issue-8-17-0-jvm]]
+===== JVM version changes needed when upgrading {ls} from 8.12.0 (or earlier)
+
+If the `jvm.options` file was modified and not overwritten with the newest version, you may see a "deserialize invocation error" message, causing the pipeline to crash. 
+Users are affected if the Persistent Queue (PQ) is enabled, and the pipeline is processing messages larger than 20MB.
+
+**Solution:** Apply the default change contained in the newer 'jvm.options' file, as seen in this https://github.com/elastic/logstash/blob/main/config/jvm.options#L66-L82[example].
```

Comment:
I understand this was just an in-place refactor, but we should link to the released tag's docs (which are static) instead of the docs on `main` (which is ever-evolving)

```suggestion
**Solution:** Apply the default change contained in the newer 'jvm.options' file, as seen in this https://github.com/elastic/logstash/blob/v8.17.0/config/jvm.options#L74-L90[example].
```